explorer/explorerroutes.go

```diff
@@ -2040,11 +2048,15 @@ func (exp *explorerUI) StatsPage(w http.ResponseWriter, r *http.Request) {
 // commonData grabs the common page data that is available to every page.
 // This is particularly useful for extras.tmpl, parts of which
 // are used on every page
-func (exp *explorerUI) commonData() *CommonPageData {
+func (exp *explorerUI) commonData(r *http.Request) *CommonPageData {
 	tip, err := exp.blockData.GetTip()
 	if err != nil {
 		log.Errorf("Failed to get the chain tip from the database.: %v", err)
 	}
+	darkMode, ckErr := r.Cookie("dcrdataDarkBG")
+	if err != nil && err != http.ErrNoCookie {
```

Comment:
did you mean to check `ckErr`?  Also it seems `commonData` should return when `GitTip` fails (template execution would probably just fail).